package.json

```diff
@@ -38,14 +38,14 @@
   },
   "dependencies": {
     "@bufbuild/protobuf": "^1.0.0",
-    "@bufbuild/protoc-gen-es": "^1.0.0",
     "buffer-builder": "^0.2.0",
     "immutable": "^4.0.0",
     "rxjs": "^7.4.0",
     "supports-color": "^8.1.1"
   },
   "devDependencies": {
-    "@protobuf-ts/protoc": "^2.8.2",
+    "@bufbuild/buf": "^1.13.1-4",
```

Comment:
does this need to be installed in ci.yml?

Reply:
I think once https://github.com/sass/dart-sass-embedded/pull/146 lands it'll pick it up from Node's internal PATH, but we'll see.